package.json

```diff
@@ -11,6 +11,7 @@
     "cookie": "^0.1.2",
     "ejs": "^2.3.1",
     "extend": "^2.0.1",
+    "json2csv": "^3.3.0",
```

Comment:
I am not 100% but I think that some form of [this](http://www.postgresql.org/docs/9.5/static/sql-copy.html) could do what you wanted without adding the json2csv module (search for csv and it should pop up)